extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -115,20 +115,31 @@
   white-space: nowrap !important;
 }
 
-.i-amphtml-lbv[gallery-view] .i-amphtml-lbv-gallery {
-  display: flex;
-  flex-wrap: wrap;
-  align-content: flex-start;
+@media screen and (min-width: 1024px) {
+  .i-amphtml-lbv[gallery-view] .i-amphtml-lbv-gallery {
+    display: grid;
+    grid-template-columns: repeat(4, 246px);
+    grid-template-rows: repeat(auto-fill, 246px);
+    justify-content: center;
+    grid-gap: 10px;
+  }
+}
+
+@media screen and (max-width: 1023px) {
```

Comment:
For media queries, it's typical to stick with `min-width` and have the base condition not in a media query. The rules inside the media queries override the defaults if needed.

```css
.my-default-and-mobile-rules {
}

@media (min-width: 640px) {
  .my-tablet-rules
  }
}

@media (min-width: 768px) {
  .my-desktop-rules
  }
}
```

Reply:
And you should factor out the duplicated properties into a common rule outside of the media queries. I'm not sure how smart our CSS minification is, but it's usually a good idea for filesize. And for readability.